modules/openapi/component-protocol/scenarios/issue-manage/components/issueFilter/model.go

```diff
@@ -43,113 +43,113 @@ type FrontendConditions struct {
 	ClosedAtStartEnd   []*int64                      `json:"closedAtStartEnd,omitempty"`
 }
 
-func generateFrontendConditionProps(fixedIssueType string, state State) FrontendConditionProps {
+func generateFrontendConditionProps(fixedIssueType string, state State, bdl protocol.ContextBundle) FrontendConditionProps {
 	conditionProps := []filter.PropCondition{
 		{
 			Key:         PropConditionKeyIterationIDs,
-			Label:       "迭代",
-			EmptyText:   "全部",
+			Label:       bdl.I18nPrinter.Sprintf("Sprint"),
```

Comment:
iteration?sprint?

Reply:
The front-end internationalization that does not use componentization uses sprint